index.js

```diff
@@ -1,10 +1,10 @@
 import { NativeModules } from 'react-native';
-let SailthruMobile = NativeModules.RNSailthruMobile;
+let Marigold = NativeModules.RNMarigold;
```

Comment:
We'll need to add the `EngageBySailthru` and `MessageStream` classes here as well and separate out the bahaviour.